aristanetworks/ansible-avd
# Doc(eos_designs): Update custom_structured_config doc following merge changes in 4.0.0

## Change Summary

Update documentation related to custom_structured_config feature following the change of the default merge strategy from `replace` to `append_rp` in v4.0.0

## Related Issue(s)

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
- Update examples and related notes

## How to test
Use default `append_rp` merge strategy to confirm interaction between:
- `eos_designs`'s `name_servers` vs `eos_cli_config_gen`'s `ip_name_servers`
- `eos_designs`'s `ipv4_acls` vs `eos_cli_config_gen`'s `ip_access_lists`

## Checklist

### User Checklist
- N/A

### Repository Checklist
